### PR TITLE
Change 'Working with design' section

### PR DESCRIPTION
**Pull request by AlicjaSuska**
Update to the design team handbook based on our [last week's conversation](https://docs.google.com/document/d/1xszELbG-5urpXf-FHPc1tLDMBb_PM3MTFjFSOFRHPTo/edit?usp=sharing) about Design Priorities board and managing design work.

**Changes summary**
1. 'Working with design / requesting design work' section moved below the 'Contact'. This place seems more in context.
2. Design Priorities board columns explained (especially 'Up next' and 'Coming soon' columns)
3. Added 'How to request design work' section
4. Added 'design labels' explanation (please, let me know if you agree with the difference between 'design' and 'needs-design')
Screenshot 2020-10-19 at 16 23 27

**Managing the design work**
For now, I'm considering all the designers responsible for assigning work from the 'coming soon' column. If we encounter some problems (ex. issues remaining unnoticed or workload problems), we might have to consider making one of the designers responsible for this. I can volunteer.

**Communicating the flow**
Once we agree on the changes, I would like to communicate the process in the following **company meeting** to make everyone aware of the updated approach.

### CODE REVIEW

**Review by quinnkeast — APPROVED** [review]

**Review by rrhyne — APPROVED** [review]
Looks great, thanks for updating!

File: handbook/product/design/index.md [hunk 1]
```diff
@@ -16,6 +16,46 @@ Here are a few of the things we strive to make true:
 
 - #design channel or @design-team on Slack
 
+## Working with design / requesting design work
+
+Working with design should feel fluid, you should be able to open an issue or bring up a concern in slack, and receive actionable feedback: either implementable, or a call to explore further.
+
+You should also not feel limited to words, images (sketches, or references made by editing the broswer DOM) are also encouraged. 
+
+### GitHub Design Priorities project
```
Comment (christinaforney) [inline handbook/product/design/index.md:25]:
```suggestion
### Tracking design priorities
```

File: handbook/product/design/index.md [hunk 2]
```diff
@@ -16,6 +16,46 @@ Here are a few of the things we strive to make true:
...
+
+Design Team uses this board for managing current and future work, design debt and work requests.
```
Comment (christinaforney) [inline handbook/product/design/index.md:27]:
```suggestion
The Design Team uses a [GitHub project board](https://github.com/orgs/sourcegraph/projects/66) for managing current and future work, design debt, and work requests.
```

File: handbook/product/design/index.md [hunk 3]
```diff
@@ -16,6 +16,46 @@ Here are a few of the things we strive to make true:
...
+
+Columns:
+
+- Design debt - issues connected to design debt that haven't been scheduled yet
+- **Backlog** - issues that need design work but don't have any development resources allocated and haven't been scheduled yet
+- **Coming soon** - no designer assigned yet, development resources have been allocated. Add **design deadline** to the issue, if possible
+- **Up next** - designer assigned, design work scheduled
+- In progress - ongoing design work
```
Comment (christinaforney) [inline handbook/product/design/index.md:35]:
```suggestion
- In progress - design work actively being worked on
```

File: handbook/product/design/index.md [hunk 4]
```diff
@@ -16,6 +16,46 @@ Here are a few of the things we strive to make true:
...
+- QA - design work done, designer is assisting the implementation
+- Measure success - design shipped, designer is working with the bizops team to analyse the impact
```
Comment (christinaforney) [inline handbook/product/design/index.md:37]:
```suggestion
- Measure success - design shipped, designer is working with the BizOps team to analyze the impact 
```

File: handbook/product/design/index.md [hunk 5]
```diff
@@ -16,6 +16,46 @@ Here are a few of the things we strive to make true:
...
+Columns:
+
+- Design debt - issues connected to design debt that haven't been scheduled yet
+- **Backlog** - issues that need design work but don't have any development resources allocated and haven't been scheduled yet
+- **Coming soon** - no designer assigned yet, development resources have been allocated. Add **design deadline** to the issue, if possible
+- **Up next** - designer assigned, design work scheduled
+- In progress - ongoing design work
+- QA - design work done, designer is assisting the implementation
+- Measure success - design shipped, designer is working with the bizops team to analyse the impact 
+- Done - success measured, all work done
```
Comment (christinaforney) [inline handbook/product/design/index.md:38]:
```suggestion
- Done - success measured, all work completed to scope
```

File: handbook/product/design/index.md [hunk 6]
```diff
@@ -16,6 +16,46 @@ Here are a few of the things we strive to make true:
...
+
+### GitHub design labels
+
+- **needs-design** - for design work requests - add to 'design priorities' project, add a deadline, if possible..
+- **design** - for design-oriented issues. This label should be mostly used by designers
+- **design-debt** - for issues connected to the design debt.
```
Comment (christinaforney) [inline handbook/product/design/index.md:44]:
```suggestion
- **`needs-design`**: design work requests. Add to [Design priorities project](https://github.com/orgs/sourcegraph/projects/66) and if possible add a deadline.
- **`design`**: design-oriented issues. This label should be mostly used by designers
- **`design-debt`**: issues connected to design debt.
```

File: handbook/product/design/index.md [hunk 7]
```diff
@@ -16,6 +16,46 @@ Here are a few of the things we strive to make true:
...
+If you need some design help:
+
+- add the tag **needs-design** to your GitHub issue
+- you can assign one of the designers, if you already know who will take care of the issue
+- add your issue to the **[design priorities project]**(https://github.com/orgs/sourcegraph/projects/66)
+- assign your issues to one of the columns - (see the 'GitHub Design Priorities project' section above for details)
+
+Please, make it clear how urgent the issue is (adding the deadline is encouraged) and if development resources have been allocated. It is important to focus design team's attention on the work that can be implemented soon. 
+
+Use #design Slack channel for any urgent matters or questions.
```
Comment (christinaforney) [inline handbook/product/design/index.md:57]:
```suggestion
- Add the **`needs-design`** tag to your GitHub issue
- Assign one of the designers if you know who will take care of the issue
- Add your issue to the [design priorities project](https://github.com/orgs/sourcegraph/projects/66)

Please clarify the urgency of the issue (adding a known deadline is encouraged) and if development resources have been allocated. It is important to focus the design team's attention on the work that can be implemented soon. 

Use the #design Slack channel for any urgent matters or questions.
```

File: handbook/product/design/index.md [hunk 8]
```diff
@@ -28,20 +68,10 @@ While design does adds time at the beginning of a project, it improves velocity
 
 Our design process is documented here: **[Design process](design_process.md)**
 
-## Working with design
-
-Working with design should feel fluid, you should be able to open an issue or bring up a concern in slack, and receive actionable feedback: either implementable, or a call to explore further.
-
-You should also not feel limited to words, images (sketches, or references made by editing the broswer DOM) are also encouraged.
-
 ### Figma
```
Comment (christinaforney) [inline handbook/product/design/index.md:71]:
```suggestion
## Figma
```

**Review by christinaforney — APPROVED** [review]
This is great! Thanks for writing. Added a few grammatical and formatting suggestions.

**Pull request merged**